install.sh

```diff
@@ -91,13 +91,15 @@ ensure_file_from_example $SENTRY_CONFIG_YML
 ensure_file_from_example $SENTRY_EXTRA_REQUIREMENTS
 
 echo ""
```

Comment:
I think this `echo` should go into the `if` block too :)

Reply:
Yes it should, I've changed it